tools/fast/bin/localnet/main.go

```diff
@@ -394,37 +379,11 @@ func isEmpty(name string) (bool, error) {
 	return false, err // Either not empty or error, suits both cases
 }
 
-func getSectorSize(smallSectors bool) (uint64, error) {
-	rp := repo.NewInMemoryRepo()
-	blockstore := bstore.NewBlockstore(rp.Datastore())
-	blockservice := bserv.New(blockstore, offline.Exchange(blockstore))
-
-	var proofsMode proofs.Mode
+func getProofsMode(smallSectors bool) proofs.Mode {
 	if smallSectors {
-		proofsMode = proofs.TestMode
-	} else {
-		proofsMode = proofs.LiveMode
+		return proofs.TestMode
 	}
-
-	// Not a typo
-	addr := address.NewForTestGetter()()
-
-	cfg := sectorbuilder.RustSectorBuilderConfig{
-		BlockService:     blockservice,
-		LastUsedSectorID: 0,
-		MetadataDir:      "",
-		MinerAddr:        addr,
-		ProofsMode:       proofsMode,
-		SealedSectorDir:  "",
-		StagedSectorDir:  "",
-	}
-
-	sb, err := sectorbuilder.NewRustSectorBuilder(cfg)
-	if err != nil {
-		return 0, err
-	}
-
-	return sb.GetMaxUserBytesPerStagedSector()
+	return proofs.LiveMode
```

Comment:
The simplification in this file is impressive!

Reply:
Building a SectorBuilder is pretty tough in most parts of the application